id_infrastructure/firestore_uuid_table.py

```diff
@@ -242,10 +245,32 @@ def get_all_mappings(self):
         :rtype: dict
         """
         self._mappings_cache = {}
-        for mapping in self._client.collection(f"tables/{self._table_name}/mappings").get():
+        for mapping in self._mappings_ref().get():
             self._mappings_cache[mapping.id] = mapping.get(_UUID_KEY_NAME)
         return self._mappings_cache.copy()
 
     @staticmethod
     def generate_new_uuid(prefix):
         return prefix + str(uuid.uuid4())
+
+    def delete_mappings(self, mappings, regexp):
+        """
+        Deletes the mappings specified.
+
+        :param mappings: Dictionary of data -> uuid
+        :type mappings: dict of str -> str
+        :param regexp: Regular expression object.
+        :type regexp: re.Pattern
+        """
+        for mapping_id in mappings.keys():
+            log.warning(f"Deleting mapping id `{mapping_id}`...")
+            mapping_ref = self._mappings_ref().document(mapping_id)
+            
+            mapping = mapping_ref.get()
```

Comment:
Can you call this something like database_mapping or firestore_mapping to distinguish it from the mapping that was passed to the function?